scripts/setup-database-and-test.sh

```
@@ -0,0 +1,402 @@
+DB_NAME_SYS="${DB_NAME_SYS:-alphaloop_sys}"
+DB_NAME_MARKET="${DB_NAME_MARKET:-alphaloop_market}"
+
+echo -e "${BLUE}🗄️ AlphaLoop Database Setup and Service Testing${NC}"
+echo "================================================"
+echo ""
+
+echo "Starting database setup and service testing..."
+echo ""
+
+# Function to generate SQL from YAML on-the-fly
+generate_sql_from_yaml() {
```

Shellcheck (0.10.0)

[error] Couldn't parse this brace group. Fix to allow more checks.

(SC1073)